docs/flows-format.adoc

```diff
@@ -191,6 +191,12 @@ The "Cardinality" column gives information about the implied metric cardinality
 | `flow_layer`
 | no
 | fine
+| `NetworkEvents`
+| string
+| Network events flow monitoring
+| `network_events`
+| no
+| avoid
```

Comment:
@msherif1234 I'm curious about the cardinality impact if we wanted to use in prom. How are the event messages built? What's the variable parts?

Reply:
network events are unique strings created for the different ovn events not sure what u mean with how are the event messages built ?

Reply:
if the string is always like "network policy $np has allowed this traffic", with $np being any network policy name, the cardinality is OK - the number of values that this message can have is equal to the number of network policies in the cluster.
But if the message of more variable parts (let's say, like some UUID of some sort for instance) then it's too high cardinality to use that in metrics.

Reply:
of course I get that it's not only about network policies it can be other kinds of events, but that's still something relatively well bounded afaict

Reply:
cc @npinaeva too , in case you have some thoughts on the matter ... I want to figure out if the network event field would be a good candidate for a prometheus label or if it has too high cardinality

Reply:
(actually, this is a string built out of a list of events, right? if so never mind, it won't work...)

Reply:
the network events is list of events and each event will have the policy or adm or firewall name and the action for exaample

![admin_policy_network_events](https://github.com/user-attachments/assets/4b00e92f-1556-445b-ac31-dbb1f6100ad0)
![egress_fw_network_events](https://github.com/user-attachments/assets/63848256-1d17-409b-960e-ca2bfe6bb59f)
![ingr_egr_network_events](https://github.com/user-attachments/assets/c567ff14-58a2-41ef-949f-f8e980867bb4)